README.md

```diff
@@ -97,10 +97,13 @@ This will create a new project in Eclipse and download a copy of this project in
   - Choose <project_directory>/style/intellij-java-google-style.xml, and hit OK a few times
 - Run stuff:
   - FUSE has some IntelliJ run configurations checked into the repository under .idea/runConfigurations. These run configs should be automatically imported by IntelliJ and listed in a drop-down in the top-right of the screen. Next to the drop-down are buttons to run the selected run configuration, to run in debug mode, or to run with coverage measurement. If the run configs are automatically found, they may need to be manually imported.
-  - The Protobuf files for this project are generated by a Maven plugin.  Before using the below run configurations, build the project with maven using mvn compile or a similar command to generate the Protobuf and avoid compilation errors.
+  - #### Generate Protobuf files
```

Comment:
there's a similar section for eclipse -- can you make the same updates there, please? Or maybe pull out "generate protobuf files" to its own section nearer the top of the page so it's easier to notice -- open to suggestions on placement.

Reply:
I clarified in the eclipse section.  I've played around with making it it's own section on top but I couldn't find somewhere that's not awkward.  I'm open to suggestions though.